lib/content-services/src/lib/search/components/search-number-range/search-number-range.component.ts

```diff
@@ -102,7 +116,12 @@ export class SearchNumberRangeComponent implements SearchWidget, OnInit {
             const value = this.formatString(this.format, map);
 
             this.context.queryFragments[this.id] = `${this.field}:${value}`;
-            this.context.update();
+            this.context.filterRawParams[this.id] = this.context.filterRawParams[this.id] ?? {};
```

Comment:
I think you can store that part: this.context.filterRawParams[this.id] in some const too to avoid repeating getting the same object multiple times below. Even when stored in const then assigning to fields should work through reference